transport/src/main/java/io/zeebe/transport/impl/AtomixServerTransport.java

```diff
@@ -68,9 +65,11 @@ public void close() {
   public ActorFuture<Void> subscribe(final int partitionId, final RequestHandler requestHandler) {
     return actor.call(
         () -> {
+          final var topicName = topicName(partitionId);
+          LOG.trace("Subscribe for topic {}", topicName);
```

Comment:
Maybe wrp in `ìsTraceEnabled()` also at other places in this class

Reply:
actually this is done by ZbLogger as far as I know

Reply:
No, it is not. ZBLogger will prevent it from being logged (which ordinary loggers do anyways), but still the string will need to be composed, which can be expensive.

In newer logging implementation you often find lambdas:
`LOG.trace(() -> .calculateLongString())` which circumvent this once and for all. But these methods aren't available in `Logger`

Reply:
ok get what you mean thanks